src/lib/components/side-nav.svelte

```diff
@@ -96,6 +106,25 @@
         <div class="nav-title">{translate('feedback')}</div>
       </NavRow>
     </slot>
+    <NavRow {isCloud} handleClick={() => ($labsMode = !$labsMode)}>
+      <NavTooltip right text={labsHoverText}>
+        <button
+          aria-label={labsText}
```

Comment:
With the recent changes we no longer need this (it's currently announcing the text twice)

```suggestion

```